salter.sh

```diff
@@ -115,7 +115,7 @@ pkg-add() {
                  /usr/bin/dnf install -y --best --allowerasing ${PACKAGES} || exit 1
              elif [ -f "/usr/bin/yum" ]; then
                  /usr/bin/yum update -y || exit 1
-                 /usr/bin/yum install -y ${PACKAGES} || exit 1
+                 /usr/bin/yum install -y ${PACKAGES} --skip-broken || exit 1
```

Comment:
This could potentially introduce more problems ?

Reply:
It's a huge risk on `yum update` but probably okay for `yum install <packages>`.  OS scope is Cent/Fedora.  Package scope is  whatever`PACKAGES` variable contains which is following (+ dependencies):
```
             if [ -f "/usr/bin/dnf" ]; then
                 PACKAGES="--best --allowerasing git wget redhat-rpm-config"
             elif [ -f "/usr/bin/yum" ]; then
                 PACKAGES="epel-release git wget redhat-rpm-config"
```
Maybe I should just get rid of `git` and assume its installed?

Reply:
Apologies - thats a different PR - I've fixed the commit now - thats gone.